lib/python/treadmill_aws/cli/admin/aws/image.py

```diff
@@ -29,19 +31,77 @@ def image():
     def _list():
         """List images"""
         ec2_conn = awscontext.GLOBAL.ec2
-        _account = awscontext.GLOBAL.sts.get_caller_identity().get('Account')
         images = ec2client.list_images(ec2_conn, owners=[_account])
         cli.out(formatter(images))
 
-#   This is a create API dummy skelleton
-    @image.command(name='create')
-    @click.argument('image_id')
+    @image.command(
+        name='create'
+    )
+    @click.option(
+        '--userdata', required=True,
+        help='EC2 Userdata <path>'
+    )
+    @click.option(
+        '--base-image-id', required=True,
+        help='EC2 AMI ID'
+    )
+    @click.option(
+        '--image-name', required=True,
+        help='EC2 AMI Name Tag'
+    )
+    @click.option(
+        '--image-version', required=True,
+        help='EC2 AMI Version Tag'
+    )
+    @click.option(
+        '--image-instance-type', default='t2.medium',
```

Comment:
is there any reason that this should be parameter? we are using it for snapshot only, it will never be running "real" app. 

probably ok to have it for now.